projects/packages/forms/src/dashboard/components/layout/index.js

```diff
@@ -1,26 +1,19 @@
 import { JetpackFooter } from '@automattic/jetpack-components';
+import // __experimentalHStack as HStack, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+'@wordpress/components';
```

Comment:
This seems odd. You're not actually importing anything from `@wordpress/components` here.